examples/connext_dds/dynamic_data_request_reply/c++11/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# (c) 2022 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+cmake_minimum_required(VERSION 3.11)
+project(rtiexamples-dynamic-data-request_reply)
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
+)
+
+# Find the RTI Connext DDS libraries
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.0"
+        REQUIRED
+        COMPONENTS
+            core
+            messaging_api
+    )
+endif()
```

Comment:
So there is [a problem](https://github.com/rticommunity/rticonnextdds-examples/runs/6492737972?check_suite_focus=true#step:11:43) here. Not really your problem. The thing is, that when we compile all the `examples/connext_dds` directory [we already call to `find_package(RTIConnextDDS`](https://github.com/rticommunity/rticonnextdds-examples/blob/master/examples/connext_dds/CMakeLists.txt#L16-L21). That time, we don't search for the `messaging_api` compoment. Then, when it arrives to this directory, as RTIConnextDDS is already found, it will not search for it. So you can remove the `if/endif` calls, and leave it like [this routing service example](https://github.com/rticommunity/rticonnextdds-examples/blob/master/examples/routing_service/file_adapter/c%2B%2B11/CMakeLists.txt#L28-L34).

```suggestion
find_package(RTIConnextDDS
    "6.1.0"
    REQUIRED
    COMPONENTS
        core
        messaging_api
)
```

Reply:
@lulivi Hello, I've made the above suggested change and pushed another commit. Could you please check again and merge the PR?